pipeline/filters/lua.md

```diff
@@ -180,7 +176,22 @@ pipeline:
 {% endtab %}
 {% endtabs %}
 
-#### Environment variable processing
+### Number Type
+
++Lua treats number as double. It means an integer field (e.g. IDs, log levels) will be converted double. To avoid type conversion, The `type_int_key` property is available.
```

Comment:
```suggestion
Lua treats numbers as a double type.
It means an integer field (e.g. IDs, log levels) will be converted to a double.
To avoid type conversion, the `type_int_key` property is available.
```